src/project-tests/survey-form-tests.js

```diff
@@ -116,6 +116,13 @@ export default function createSurveyFormTests() {
 
             it('12. I can select an option from a dropdown that has corresponding id="dropdown".', function() {
                 var dropdown = document.getElementById('dropdown');
+				var dropdownid = dropdown.getAttribute('id');
+				var dropdowntag = dropdown.tagName;
+				if (dropdowntag === 'DATALIST') {
+					var inputsibling = dropdown.previousElementSibling;
```

Comment:
@no-stack-dub-sack I didn't know we were doing ES6 with this project which is why I didn't suggest using `let`. I don't see`let` used anywhere else. I'm comfortable with ES6 and use it whenever I can, so some direction on what we should use would be welcome.